Doc/reference/compound_stmts.rst

```diff
@@ -308,9 +308,10 @@ from a function that handled an exception.
    statement: break
    statement: continue
 
-The optional :keyword:`else` clause is executed if and when control flows off
-the end of the :keyword:`try` clause. [#]_ Exceptions in the :keyword:`else`
-clause are not handled by the preceding :keyword:`except` clauses.
+The optional :keyword:`else` clause is executed if no exception was raised, and
+if no :keyword:`return`, :keyword:`continue`, or :keyword:`break` statement was
```

Comment:
IMO the second "if" in this sentence needs to be removed for correctness, since the `else` clause will execute only if *both* conditions hold.

Reply:
That was the intention of my wording 😅

I'll change it, thanks for reviewing :)